saleor/graphql/payment/mutations/transaction/transaction_update.py

```diff
@@ -97,33 +97,49 @@ def check_can_update(
 
     @classmethod
     def validate_transaction_input(
-        cls, instance: payment_models.TransactionItem, transaction_data
+        cls,
+        instance: payment_models.TransactionItem,
+        transaction_data,
+        transaction_event_data
```

Comment:
Maybe it's just a small nitpick - the first mutation uses `transaction` and `transaction_event` names, while this one has different ones. It would be nice to use the same pattern for consistency as I think these are the same objects.